sdk/core/Azure.Core/src/Pipeline/RetryPolicy.cs

```diff
@@ -283,6 +286,11 @@ private TimeSpan CalculateNextDelayInternal(HttpMessage message)
                 delay = serverDelay;
             }
 
+            if (_delayStrategy != null)
```

Comment:
Should `CalculateNextDelayInternal` be implemented as a default `DelayStrategy`? If so, we could set it in the ctor based on the RetryOptions (`ConstantDelayStrategy`, `ExponentialDelayStrategy`, etc)

Reply:
Yeah it can be - I was mostly just trying to get it to build to generate an API